app/models/user.rb

```diff
@@ -1,2 +1,28 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  # has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :posts, foreign_key: 'author_id', counter_cache: true
+  has_many :comments, dependent: :destroy
+  has_one :like, dependent: :destroy
```

Comment:
- 👋🏻  Hello @enocol! Awesome job on this PR and getting the associations set up! 🌟 However, for this particular milestone, according to the ERD diagram, a user can have multiple likes. So, let's switch from `has_one :like, dependent: :destroy ` to the more fitting `has_many :likes`. Keep up the great work! 🚀